README.md

```diff
@@ -3,3 +3,5 @@
 setup: install [minipro](https://gitlab.com/DavidGriffith/minipro)
 
 flash me with `make generate && minipro -p ATF22V10C -w GAL22V10_fifo_buf.jed`
+
+https://www.proto-advantage.com/store/product_info.php?products_id=2200264
```

Comment:
I like the idea of this repo being notes & such about different hardware-y things, maybe we make a `NOTES.md` for that stuff? And/or, drop a touch of context in here for this?